### ISSUE TITLE
Exclude packages based on regexp

### ISSUE THREAD

**Issue by adrien-n** [mclemenceau/visual-excuses#12]
GHC/Haskell/OCaml/Coq/python clusters are often large, blocked by ecosystem changes, and can be of lower priority than some more central packages. Moreover, they're sometimes best left to their respective (Debian) team which is often already working on the migration. Showing them reduces visibility of other packages. I think this could be solved by having a command-line switch to skip nodes and edges from/to packages which name matches a given regexp. For instance:

    --exclude ".*(haskell|ghc|ocaml|coq).*

Ideally you could pass several --exclude which would accumulate.

An extra step would be to support something predefined filters which could be more complex regexps because ".*ocaml.*" is probably not the best filter for OCaml packages (deciding on the exact filters would take some time and would probably be best done once --exclude is in place and we can see the results we get for every regexp).
